app/dashboard/views.py

```diff
@@ -250,9 +250,19 @@ def receive_tip(request):
 @csrf_exempt
 @ratelimit(key='ip', rate='1/m', method=ratelimit.UNSAFE, block=True)
 def send_tip_2(request):
-    """Handle the second stage of sending a tip."""
-    username = request.session.get('handle')
-    primary_email = request.session.get('email', '')
+    """Handle the second stage of sending a tip.
+
+    TODO:
+        * Convert this view-based logic to a django form.
```

Comment:
i need to learn more about django forms :P

Reply:
:P